helium-core/src/main/java/com/joaquimverges/helium/core/retained/RetainedPresenters.kt

```diff
@@ -5,22 +5,19 @@ import android.arch.lifecycle.ViewModelProviders
 import android.support.v4.app.Fragment
 import android.support.v4.app.FragmentActivity
 
-object RetainedPresenters {
+/**
+ * Convenient way to access retained Presenters.
+ * Presenters instantiated this way will be persisted across configuration changes,
+ * and only destroys when the activity is really destroyed.
+ *
+ * @param activity the Activity to bind this Presenter to
+ * @param clazz the class of the Presenter to instantiate.
+ */
+inline fun <reified P : ViewModel> FragmentActivity.getRetainedPresenter(): P = ViewModelProviders.of(this).get(P::class.java)
 
-    /**
-     * Convenient way to access retained Presenters.
-     * Presenters instantiated this way will be persisted across configuration changes,
-     * and only destroys when the activity is really destroyed.
-     *
-     * @param activity the Activity to bind this Presenter to
-     * @param clazz the class of the Presenter to instantiate.
-     */
-    fun <P : ViewModel> get(activity: FragmentActivity, clazz: Class<P>): P = ViewModelProviders.of(activity).get(clazz)
+/**
+ * @param factory optional lambda to construct the Presenter yourself if it has non-empty constructors.
+ */
+inline fun <reified P : ViewModel> FragmentActivity.getRetainedPresenter(noinline factory: (Class<P>) -> P): P = ViewModelProviders.of(this, PresenterFactory(P::class.java, factory)).get(P::class.java)
 
-    /**
-     * @param factory optional lambda to construct the Presenter yourself if it has non-empty constructors.
-     */
-    fun <P : ViewModel> get(activity: FragmentActivity, clazz: Class<P>, factory: (Class<P>) -> P): P = ViewModelProviders.of(activity, PresenterFactory(clazz, factory)).get(clazz)
-
-    fun <P : ViewModel> get(fragment: Fragment, clazz: Class<P>): P = ViewModelProviders.of(fragment).get(clazz)
-}
\ No newline at end of file
+inline fun <reified P : ViewModel> Fragment.getRetainedPresenter(): P = ViewModelProviders.of(this).get(P::class.java)
```

Comment:
not sure why i didn't add a Fragment version of this with a factory. Mind adding it ? 🙏